feat: add summit 2022 banner

k8isdead opened the pull request:
This PR brings a banner for the eBPF 2022 summit

[Preview link](https://deploy-preview-190--ebpfio.netlify.app/)

xmulligan commented on src/layouts/index.js:
```diff
@@ -10,122 +10,254 @@ import "./menu-icon.scss";
+      className="introDisclaimer-button"
+      href="https://www.ebpf.io/summit-2022"
+    >
+      Learn More
```
```suggestion
      Register Now!
```

k8isdead replied:
done!

k8isdead commented:
@xmulligan I've updated the button text

@qmonnet thanks for the review!
The formatting changes are my bad, I've failed to notice these linter-introduces changes before commiting, won't do so next time (or will add them in a separate PR). 

Please note that we have also updated the banner mobile layout, so that the date would still be visible on small screens.
